src/app/shared/components/export-csv/export-config-base.service.ts

```diff
@@ -34,5 +35,5 @@ export abstract class ExportConfigService {
     return result;
   }
   
-  public abstract getConfig(): ExportItem[];
+  public abstract getConfig(param?: unknown): ExportItem[];
```

Comment:
@diamond-darrell why unknown instead of the already imported ViewConfig?

Reply:
in this case, we don't limit the param only to the viewConfig, we need to set it abstract in the base class, so other configs can define its own params if needed.
For example here https://github.com/kaltura/analytics-front-end/blob/master/src/app/modules/user/user-export.config.ts#L13